crates/lsp/src/lsp.rs

```diff
@@ -540,7 +540,7 @@ impl LanguageServer {
                         completion_item: Some(CompletionItemCapability {
                             snippet_support: Some(true),
                             resolve_support: Some(CompletionItemCapabilityResolveSupport {
-                                properties: vec!["additionalTextEdits".to_string()],
+                                properties: vec!["documentation".to_string(), "additionalTextEdits".to_string()],
```

Comment:
For some reason, logic to handle resolving documentation for Completionitem was there, just not included in the resolve_providers of the completion items. This caused no documentation to show for CompletionItem's for LSPs that require resolving completionItems for documentation. 

Not sure why it was omitted as it seems to work fine? Should be supported nonetheless based on the [spec](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_completion):

> If computing full completion items is expensive, servers can additionally provide a handler for the completion item resolve request (‘completionItem/resolve’.  A typical use case is for example: the textDocument/completion request doesn’t fill in the documentation property for returned completion items since it is expensive to compute

In eithercase, I pushed a change if it's fine, otherwise will revert.

Reply:
Work fine was something I should've tested, it doesn't. I've removed this commit. Unfortunately it would mean lack of documentation on CompletionItem's, but better than to have a buggy experience..